src/Bindings/ManualBindings_World.cpp

```diff
@@ -463,6 +463,52 @@ static int tolua_cWorld_DoWithPlayerByUUID(lua_State * tolua_S)
 
 
 
+static int tolua_cWorld_DoWithNearestPlayer(lua_State * tolua_S)
+{
+	// Check params:
+	cLuaState L(tolua_S);
+	if (
+		!L.CheckParamSelf("cWorld") ||
+		!L.CheckParamUserType(2, "Vector3<double>") ||
+		!L.CheckParamNumber(3) ||
+		!L.CheckParamFunction(4) ||
+		// Params 5 and 6 are optional bools, no check for those
+		!L.CheckParamEnd(7)
+		)
+	{
+		return 0;
+	}
+
+	// Get parameters:
+	cWorld * Self;
+	Vector3d * Position;
+	double RangeLimit;
+	cLuaState::cRef FnRef;
+	bool CheckLineOfSight = true, IgnoreSpectators = true;  // Defaults for the optional params
+	L.GetStackValues(1, Self, Position, RangeLimit, FnRef, CheckLineOfSight, IgnoreSpectators);
+
+	if (!FnRef.IsValid())
+	{
+		return L.ApiParamError("Expected a valid callback function for parameter #5");
```

Comment:
This should now say parameter number 3 (it's offset by 1 from the `CheckParam` above because the "self" parameter is passed implicitly).